tasks/keepalived_install.yml

```diff
@@ -70,6 +85,7 @@
     cache_valid_time: "{{ (ansible_pkg_mgr == 'apt') | ternary(cache_timeout, omit) }}"
   tags:
     - keepalived-apt-packages
+  notify: "systemctl-daemon-reload"
 
 - name: Revert keepalived start prevention
```

Comment:
We should probably change this task:

For non systemd, the prevent start file should be absent, but for systemd the link should be restored to the service.

Reply:
My bad, it's something else, some sources ship with wrong files, and load sample files. I will submit a combined patch.